stellar/contract.js

```diff
@@ -1,40 +1,57 @@
 'use strict';
 
-const { Contract, SorobanRpc } = require('@stellar/stellar-sdk');
+const { Address, Contract, SorobanRpc } = require('@stellar/stellar-sdk');
 const { Command, Option } = require('commander');
 const { execSync } = require('child_process');
 const { loadConfig, printInfo, saveConfig } = require('../evm/utils');
-const { stellarCmd, getNetworkPassphrase, addBaseOptions, getWallet, broadcast } = require('./utils');
+const { stellarCmd, getNetworkPassphrase, addBaseOptions, getWallet, broadcast, serializeValue, addressToScVal } = require('./utils');
 const { getChainConfig, addOptionsToCommands } = require('../common');
 const { prompt } = require('../common/utils');
 require('./cli-utils');
 
 const MAX_INSTANCE_TTL_EXTENSION = 535679;
 
-async function submitOperation(chain, _, contract, operation, _args, options) {
-    const pauseOperation = operation;
+async function submitOperation(chain, _, contract, operation, options) {
```

Comment:
```suggestion
async function submitOperation(chain, _, contract, operation, options, showReturnValue = false) {
```
optionally show return value

Reply:
good to have, updated